app/javascript/data/lockReasonOptions.js

```diff
@@ -0,0 +1,6 @@
+export const lockReasonOptions = [
```

Comment:
I think these are simple enough to just render in the dropdown in the userModal component and this file is not needed

Reply:
Got it! [c95ae087c5e7e8c5986f457d5233f67240c1c6d6](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/c95ae087c5e7e8c5986f457d5233f67240c1c6d6)